js/store.js

```diff
@@ -43,7 +51,15 @@ const store = {
     }
 
     const websites = clone(this._websites);
-    websites[hostname] = data;
+
+    if(!websites[hostname]) {
+      websites[hostname] = {};
+    }
+
+    for(const key in data) {
```

Comment:
oh yes, lot of inconsistencies here for `if` and `for` statements. sending a fix.